newswires/client/src/WiresCards.tsx

```diff
@@ -21,62 +28,98 @@ export const WireCardList = ({ wires }: { wires: WireData[] }) => {
 		[currentState.location],
 	);
 
+	const handleSelect = useCallback(
+		(id: number) => {
+			pushState({ location: `item/${id}`, params: currentState.params });
+		},
+		[currentState.params, pushState],
+	);
+
 	return (
 		<div>
-			<EuiFlexGroup gutterSize="s" direction="column">
-				{wires.map((wire) => (
-					<WirePanel
-						wire={wire}
-						key={wire.id}
-						onClick={() =>
-							pushState({
-								location: `item/${wire.id}`,
-								params: currentState.params,
-							})
-						}
-						selected={selectedWireId == wire.id.toString()}
+			<EuiTable tableLayout="auto">
+				<EuiTableHeader
+					css={css`
+						${euiScreenReaderOnly()}
+					`}
+				>
+					<EuiTableHeaderCell>Headline</EuiTableHeaderCell>
+					<EuiTableHeaderCell>Version Created</EuiTableHeaderCell>
+				</EuiTableHeader>
+				{wires.map(({ id, content }) => (
+					<WireDataRow
```

Comment:
there are console warnings `Warning: validateDOMNesting(...): <tr> cannot appear as a child of <table>. Add a <tbody>, <thead> or <tfoot> to your code to match the DOM tree generated by the browser` - Is there a `EuiTableBody` component or something similar that needs to be added here?

Reply:
good call, thanks! I'd checked the DOM structure (which was valid) but didn't realise that Eui had this more stringent requirement. The docs aren't great but as you suggested there is indeed an `EuiTableBody` component and adding it seems to have got rid of the warning 👍👍